sensor/kubernetes/eventpipeline/pipeline_impl.go

```diff
@@ -0,0 +1,83 @@
+package eventpipeline
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/sensor/common"
+	"github.com/stackrox/rox/sensor/kubernetes/eventpipeline/output"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+type eventPipeline struct {
+	listener common.SensorComponent
+	output   output.Queue
+
+	eventsC chan *central.MsgFromSensor
+	stopSig *concurrency.Signal
+}
+
+// Capabilities implements common.SensorComponent
+func (*eventPipeline) Capabilities() []centralsensor.SensorCapability {
+	return nil
+}
+
+// ProcessMessage implements common.SensorComponent
+func (*eventPipeline) ProcessMessage(msg *central.MsgToSensor) error {
+	return nil
+}
+
+// ResponsesC implements common.SensorComponent
+func (p *eventPipeline) ResponsesC() <-chan *central.MsgFromSensor {
+	return p.eventsC
+}
+
+// Start implements common.SensorComponent
+func (p *eventPipeline) Start() error {
+	log.Info("STARTING EVENT PIPELINE")
```

Comment:
Remove lingering logs as well. Maybe we could add some real logging that would be helpful on debugging.

Reply:
Done